lib/trim-off-newlines.js

```diff
@@ -0,0 +1,4 @@
+export default function trimOffNewlines(text) {
+	const regex = '/^[\r\n]+|[\r\n]+$/g';
```

Comment:
Make it a regex literal and just put it inline in the replace call.

Reply:
I've updated the PR with your suggestion 👍